src/main/java/com/iota/iri/zmq/MessageQ.java

```diff
@@ -41,13 +67,22 @@ private MessageQ(int port, String ipc, int nthreads, boolean enabled) {
         }
     }
 
+    /**
+     * Publishes an event to the queue
+     *
+     * @param message message body, prepended by topic string
+     * @param objects objects to be rendered as part of the message
```

Comment:
not sure "to be rendered" is the best term, but I don't have a better one now.
from String.format: 
```
args - Arguments referenced by the format specifiers in the format string
```